Dockerfile.cmd

```diff
@@ -18,7 +18,8 @@ WORKDIR /go/src/open-match.dev/open-match
 
 ARG IMAGE_TITLE
 
-RUN make "build/cmd/${IMAGE_TITLE}"
+RUN mkdir -p build/cmd/${IMAGE_TITLE}
```

Comment:
The make command doesn't only run go build.  It also takes care of copying and other arbitrary build actions.

Reply:
Having make's way of building things an docker's way of building things is an intentional unification.  Don't revert it.

Reply:
Our Makefile has a bunch of third party dependencies. If you run `make build-images` you will be able to see the error message that I mentioned.

Reply:
I agree, but they're warnings that don't break anything, so I've been ignoring them and haven't prioritized fixing them.